### PR TITLE
feat(home): our actions section

### PR DESCRIPTION
**Pull request by paulgrym**
## Jira issue

JIRA issue: [FSU-119](https://klau.atlassian.net/browse/FSU-119)

## Describe your changes
Created section about foundation actions. To improve: tablet and mobile view od TextBanner and Card components.

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/HomePage/components/Actions/Actions.styled.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    text-align: left;
+    align-items: flex-start;
+  }
+`
+
+export const TextContainer = styled.div`
+  max-width: 596px;
+`
+export const CardsContainer = styled.div`
+  font-family: ${({ theme }) => theme.fontFamily};
```
Comment (im-klau) [inline packages/frontend/app/templates/HomePage/components/Actions/Actions.styled.js:21]: you can remove this line, I know the font won't be the one that's available in our project but it's a bug from the Title component which will be fixed in another PR

I'd approve this one, even with the wong font family :D

File: packages/frontend/app/templates/HomePage/components/Actions/Actions.styled.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+`
+export const BannerContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 60px;
+  align-items: center;
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    text-align: left;
+    align-items: flex-start;
+  }
```
Comment (im-klau) [inline packages/frontend/app/templates/HomePage/components/Actions/Actions.styled.js:14]:
```suggestion
  ${({ theme }) => theme.breakpoints.tablet} {
    text-align: left;
    align-items: flex-start;
  }
```

File: packages/frontend/app/templates/HomePage/components/Actions/Actions.styled.js (hunk 1 above)
Comment (paulgrym) [reply packages/frontend/app/templates/HomePage/components/Actions/Actions.styled.js:21]: Without this line also List has wrong font 😢

**Review by AmandaKr — APPROVED** [review]
Good job! Code is super clean 🥇

**Review by Ebmili — APPROVED** [review]
Section is ready to go :) Looks great
